contrib/packs/actions/pack_to_git_url.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import requests
+import json
+import getopt
+import argparse
+import os
+import yaml
+
+from getpass import getpass
+from st2actions.runners.pythonrunner import Action
+
+debug = False
+
+class Packs_pack_to_git_url(Action):
+    def __init__(self, config=None):
+        super(Packs_pack_to_git_url, self).__init__(config=config)
+
+    def expand_repo_details(self, pack_to_expand, item):
+        if self.config["repositories"][pack_to_expand].has_key(item):
+            return self.config["repositories"][pack_to_expand][item]
+        else:
+            if self.config["repositories"]["default"].has_key(item):
+                return self.config["repositories"]["default"][item]
+            else:
+                return None
+
+    def run(self, pack_to_expand):
+        exit_code = 0
+
+        # Set up the results object
+        results = {}
+        results['pack_to_expand'] = pack_to_expand
+
+        if pack_to_expand.count('/') == 0:
```

Comment:
I believe we already have a similar functionality in one of the existing packs actions, wonder if anything can be re-used here.